src/sysext/sysext.c

```diff
@@ -2082,6 +2130,19 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_no_reload = true;
                         break;
 
+                case ARG_MUTABLE:
+                        if (streq(optarg, "auto"))
+                                arg_mutable = MUTABLE_AUTO;
+                        else if (streq(optarg, "import"))
+                                arg_mutable = MUTABLE_IMPORT;
+                        else {
+                                r = parse_boolean(optarg);
+                                if (r < 0)
+                                        return log_error_errno(r, "Failed to parse argument to --mutable: %s", optarg);
```

Comment:
we have this rule that cmdline switches (and conf file settings) that take a parameter are suffixed with `=` in log messages and int docs. Thus please write `--mutable=` here

Reply:
Done.